demo/api/schema.gql

```diff
@@ -1148,13 +1135,19 @@ input ProductInput {
   discounts: [ProductDiscountsInput!]! = []
   articleNumbers: [String!]! = []
   dimensions: ProductDimensionsInput
-  packageDimensions: ProductPackageDimensionsInput
+  packageDimensions: ProductPackageDimensionsInput!
```

Comment:
@nsams I simply removed the @Field decorator, should I really remove the field completely to also get rid of the requirement for input?

Reply:
the problem is the input, `@Field` is responsible for output

you could use `@CrudField({ input: false })`

Reply:
Delete it completwly